hw/ip/aon_timer/data/aon_timer.hjson

```diff
@@ -216,6 +216,9 @@
           desc: "Raised if the watchdog timer has hit the bark threshold",
         }
       ]
+      tags: [// register implemented in clk_aon domain and does not update according to
+             // normal timing
+        "excl:CsrAllTests:CsrExclWriteCheck"],
```

Comment:
This change seems to be unrelated to csrng. It should go into a separate PR.

Reply:
Done - #5373

Reply:
Is this still needed after the fix by @weicaiyang ?

Reply:
procedurally, this is what has happened: I added this tag in this PR, but @weicaiyang created a separate PR to fix this, so I rebased to pick his up, got a conflict, which I merged by choosing his fix. This is why the change shows up I guess in this PR, but it is a consequence of a merge. I can remove the merge, but this is sort of a git question in general.

Reply:
Yes, the idea is not to use merge commits but just clean rebases. The commit of yours implementing that change then becomes empty and can be dropped. This has now happend, all good.